data/language/en-GB.txt

```diff
@@ -3649,6 +3649,8 @@ STR_6392    :Could not find {STRING} at this path.
 STR_6393    :Objective Selection
 STR_6394    :Objective
 STR_6395    :Maintenance
+STR_6396    :Disable screensaver
+STR_6397    :{SMALLFONT}{BLACK}If checked, screensaver functionality will be inhibited while OpenRCT2 is running.
```

Comment:
Perhaps this should be amended to mention power saving as well, considering that was the original issue.

Reply:
Maybe something like this:
STR_6397    :{SMALLFONT}{BLACK}If checked, screensaver functionality will be inhibited while OpenRCT2 is running.{NEWLINE}Unchecking allows screen saver to run and (if in system enabled) turn off the monitor.

Reply:
and the main option string:
STR_6396    :Disable screensaver and monitor power saving mode

Reply:
I was thinking among the lines of:
```suggestion
STR_6396    :Disable screensaver and monitor power saving
STR_6397    :{SMALLFONT}{BLACK}If checked, screensaver and other monitor power saving features will be inhibited while OpenRCT2 is running.
```
It would be good to wait for some input on this from others, I think.

Reply:
Meanwhile screenshot to have a look at those changes:
![OpenRCT2_2020-10-20 14-32-15](https://user-images.githubusercontent.com/36161245/96586398-3f11ba00-12e1-11eb-9204-7eff7367405a.png)

Reply:
I prefer the look of the first of the two screenshots. (Translations will look different, anyway, of course...)

Reply:
I'll wait with commit till discussion regarding `ToggleScreenSaverLock();` will be resolved, to stop commit spam :)